lib/index.js

```diff
@@ -279,14 +280,20 @@ OniyiCache.prototype.put = function put(data, cb) {
   const saveIt = self.redisClient.multi();
 
   const selectedData = _.pick(data, ['response', 'raw', 'parsed']);
-  saveIt.hmset(key, selectedData);
+  // if we want to update expireAt only, redis will throw an error
```

Comment:
if you only want to update the expires info for the cache, use a different redis command.

Reply:
make sense.